client/components/tinymce/plugins/simple-payments/dialog/form.jsx

```diff
@@ -204,28 +229,95 @@ class ProductForm extends Component {
+							{ translate( 'Make this product a recurring subscription.' ) }
+						</ReduxFormFieldset>
+					) }
+					{ ! this.props.isRecurringSubscription && (
+						<div>
+							<ReduxFormFieldset name="multiple" type="checkbox" component={ CompactFormToggle }>
+								{ translate( 'Allow people to buy more than one item at a time.' ) }
+							</ReduxFormFieldset>
+							<ReduxFormFieldset
+								name="email"
+								label={ translate( 'Email' ) }
+								explanation={ translate(
+									'This is where PayPal will send your money.' +
+										" To claim a payment, you'll need a {{paypalLink}}PayPal account{{/paypalLink}}" +
+										' connected to a bank account.',
+									{
+										components: {
+											paypalLink: <ExternalLink href="https://paypal.com" target="_blank" />,
+										},
+									}
+								) }
+								component={ FormTextInput }
+							/>
+						</div>
+					) }
+					{ this.props.isRecurringSubscription && this.renderRecurringFields() }
 				</div>
 			</form>
 		);
 	}
+
+	renderRecurringFields() {
+		const { translate } = this.props;
+		return (
+			<div>
+				<QueryMembershipsConnectedAccounts />
+				<ReduxFormFieldset
+					name="stripe_account"
+					explanation={ translate( 'This is the Stripe account where the funds will end up.' ) }
+					label={ translate( 'Stripe account' ) }
+					component={ FormSelect }
+					children={ Object.values( this.props.membershipsConnectedAccounts )
+						.map( acct => (
+							<option
+								value={ acct.connected_destination_account_id }
+								key={ acct.connected_destination_account_id }
+							>
+								{ acct.payment_partner_account_id }
+							</option>
+						) )
+						.concat( [
+							<option value="create" key="create">
+								{ translate( 'Create Stripe account for me' ) }
+							</option>,
+							<option value="authorize" key="authorize">
+								{ translate( 'I already have a Stripe account' ) }
+							</option>,
+						] ) }
+				/>
+				{ this.props.isChoosingToAuthorizeStripeAccount && (
+					<Button onClick={ this.props.authorizeStripeAccount }>
+						{ translate( 'Authorize Stripe account' ) }
+					</Button>
+				) }
+				{ this.props.isChoosingToCreateStripeAccount && (
+					<div>
+						<ReduxFormFieldset
+							name="email"
+							label={ translate( 'Email' ) }
+							explanation={ translate( 'New Stripe account will be tied to this email address.' ) }
+							component={ FormTextInput }
+						/>
+					</div>
+				) }
+				<ReduxFormFieldset
+					name="renewal_schedule"
+					explanation={ translate( 'After what time should the subscription renew?' ) }
+					label={ translate( 'Renewal Schedule' ) }
+					component={ FormSelect }
+				>
+					<option value="1 week">{ translate( '1 Week' ) }</option>
+					<option value="1 month">{ translate( '1 Month' ) }</option>
+					<option value="1 year">{ translate( '1 Year' ) }</option>
```

Comment:
Hi! I've found a possible matching string that has already been translated **71** times:
`translate( 'year' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).